index.html

```diff
@@ -0,0 +1,89 @@
+      </nav>
+    </nav>
+    <nav class="nav2">
+      <div class="nav2Content">
+        <a id="bordered-link" class="nav-item" href="#">AI Compaign</a>
+        <a class="nav-item" href="#">News</a>
+        <a class="nav-item" href="#sponser">Sponser</a>
+        <a class="nav-item" href="#">Join</a>
+        <a class="nav-item" href="#mainProgram">Program</a>
+        <a class="nav-item" href="about.html">About</a>
+      </div>
+      <div class="navLogo"><a href="#"><img src="images/Logo.png" alt="logo"></a></div>
+    </nav>
+  </header>
+  <section class="opening">
+    <button class="mobile-menu" type="menu" ><img src="https://img.icons8.com/ios/50/000000/menu--v4.png"/></button>
+      <span class="navbar-toggler-close"></span>
+    </button>
+    <div class="mobile-menu-list">
+      <a class="nav-item" href="about.html">About</a>
+      <a class="nav-item" href="#mainProgram">Program</a>
+      <a class="nav-item" href="#">Join</a>
+      <a class="nav-item" href="#sponser">Sponser</a>
+      <a class="nav-item" href="#">News</a>
+      <a class="nav-item" href="#">AI Compaign</a>
+    </div>
+    <h2 class="helloText">"Hello! AI World"</h2>
+    <h1 class="title">MACHIN LEARNING<br>ONLONE COURSES<br>2022</h1>
+    <p class="textBox">“Artificial Intelligence, deep learning, machine learning... whatever you are doing if you don’t understand it, learn it. Because otherwise you are going to be a dinosaur within 3 years.” <br> ~Mark Cuban</p>
+    <p class="dateText">2022.02.07(Mon) ~ 11(Fri)</p>
+    <p class="locationText">@ AI conference</p>
+  </section>
+  <section class="mainProgram" id="mainProgram">
+    <h2 class="programTitle">Main Program</h3>
+      <div class="red-line"></div>
+    <div class="programCards">
+    </div>
+    <button class="join-btn" type="button">Join ML Online Course</button>
+    <a href="#">SEE THE WHOLE PROGRAM</a>
+  </section>
+  <section class="lecturers">
+    <h2 class="lecturerTitle">Lecturers</h2>
+      <div class="red-line"></div>
+    <div class="lecturersCards">
+    </div>
+  </section>
+  <section class="partner" id="sponser">
+    <h2 class="programTitle">Partners</h2>
+      <div class="red-line"></div>
+    <div class="partner-list">
+      <a class="partner-item">Google</a>
+      <a class="partner-item">DataRobot</a>
+      <a class="partner-item">LogicMonitor</a>
+      <a class="partner-item">OpenAI</a>
+    </div>
+  </section>
+  <section class="license" id="whiteLicense">
+    <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">
+      <img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" />
+    </a>
+    <br />
+    This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>.
```

Comment:
- [ ] According to the requirements, your footer should contain a logo aligned to the left and text aligned to the right, but currently, your footer doesn't follow this. It would be nice if you could address this issue.
- Your design:
![C3](https://user-images.githubusercontent.com/85113567/153273744-68f63db8-5fc3-4fb6-b07d-d6a8c49fd696.PNG)
- Required design:
![C4](https://user-images.githubusercontent.com/85113567/153273788-0f9806bc-6520-4d0e-a2f4-8064a9c0e09a.PNG)